qiskit_braket_provider/providers/adapter.py

```diff
@@ -197,6 +201,23 @@
     ),
     "gphase": qiskit_gates.GlobalPhaseGate(Parameter("theta")),
     "measure": qiskit_gates.Measure(),
+    "kraus": qiskit_qi.Kraus([[[1, 0], [0, 0]], [[0, 0], [0, 1]]]).to_instruction(),
```

Comment:
This does not look right. `kraus` can be customized with Kraus operators, not just the depolarizing channel